__mocks__/popper.js.js

```diff
@@ -0,0 +1,35 @@
+/**
```

Comment:
This is a global mock for `popper.js`. It now removes all `document.createRange` errors that were visible while running tests.

Reply:
I removed this locally and am not seeing any errors.

Reply:
The `console.errors` don't trigger test failures, but you can view them in CI or if you look back through the test runner.